cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -200,7 +202,7 @@ def service_request(appliance, ansible_catalog_item):
 
 @pytest.mark.tier(3)
 @pytest.mark.nondestructive
-@pytest.mark.meta(automates=[1656318, 1706989])
+@pytest.mark.meta(automates=[1652116, 1656318, 1706989])
 def test_collect_log_depot(log_depot, appliance, service_request, configured_depot, request):
```

Comment:
It would be better if you add these BZs in doc string under *BugZilla:* tag.

Reply:
I would leave that _Optional_, up to you if you want to include it

Reply:
Agree with you @john-dupuy . Added as it might be useful for some.